iceoryx_posh/test/moduletests/test_capro_service.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019 by Robert Bosch GmbH. All rights reserved.
+// Copyright (c) 2019, 2021 by Robert Bosch GmbH. All rights reserved.
```

Comment:
Can you please rework the fileheader according to the new guidelines in https://github.com/eclipse-iceoryx/iceoryx/blob/master/CONTRIBUTING.md#header?

Reply:
done